assets/styles/abstracts/mixins/_max-width.scss

```diff
@@ -0,0 +1,14 @@
+@mixin max-width($breakpoint) {
+  $raw-query: map-get($breakpoints, $breakpoint);
+
+  @if $raw-query {
```

Comment:
Would it be possible to either dramatically simplify both mixins (by removing all the redundant code and checks) or to share this redundant code in a shared mixin? I'd go for the simplification. Something in the lines of:

```
@mixin min-width($name) {
  $breakpoint: map-get($breakpoints, $name);

  @media (min-width: #{$breakpoint}) {
    @content;
  }
}
```

and

```
@mixin max-width($name) {
  $breakpoint: map-get($breakpoints, $name);

  @media (max-width: #{$breakpoint - 1px}) {
    @content;
  }
}
```

Reply:
I agree with the simplification. I just changed it.